tests/E2E/test/clients/common_client.js

```diff
@@ -885,6 +904,22 @@ class CommonClient {
           }
         }, selector, value, isXpath);
   }
+
+  /**
+   * remove attribute from selector
+   * @param selector, xpath or css selector of the element
+   * @param attributeName, attribute to remove
+   * @param isXpath, true if selector is xpath, false if css selector
+   * @param pause
+   */
+  removeAttribute(selector, attributeName, isXpath = true, pause = 0) {
```

Comment:
It seems a bit strange to perform the pause inside the `removeAttribute` 🤔 why not before calling it ?

Maybe I'm missing something here

Reply:
no need for pause in this function

Reply:
deleted